subsystems/PistonControl.java

```diff
@@ -0,0 +1,38 @@
+package org.frc5587.lib.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+
+import static edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public abstract class PistonControl extends SubsystemBase{
+    private DoubleSolenoid[] doubleSolenoidPistons;
+
+    public PistonControl(int[][] pistonIDS) {
```

Comment:
Give a brief explanation of the purpose of this class and the parameters. To autopopulate the javadoc, type `/**` and a box should appear saying "java docs" or smth, just hit enter